src/components/Send.tsx

```diff
@@ -1,4 +1,121 @@
+import api, { CreatePSBTOutputs } from '../api';
+import { Wallet } from '../types';
 
-// TODO: Send page
-export default (() => <>I'm the Send page</>) as React.SFC;
+interface DispatchProps {
+  getWallets: typeof getWallets;
+}
+
+interface ConnectProps {
+  // FIXME: typescript makes me accept possibility of nullness
+  // How can we just demand that this not be null for the component to even load?
+  activeWallet: Wallet | null;
+}
+
+type Props = DispatchProps & ConnectProps & RouteComponentProps;
+
+interface StateProps {
+  activeWallet: AppState['wallet']['activeWallet'];
+}
+
+interface LocalState {
+  outputs: CreatePSBTOutputs;
+  isSubmitting: boolean;
+  error: Error | null;
+}
+
+class Create extends React.Component<Props, LocalState> {
+  state: LocalState = {
+    outputs: [{'address': undefined, 'btc': undefined}],
+    isSubmitting: false,
+    error: null,
+  };
+  render() {
+    const { outputs } = this.state;
+
+    return (
+      <Form onSubmit={this.handleSubmit}>
+        {outputs.map((output, index) =>
+          <div>
+            <FormGroup>
+              <Label>Address</Label>
+              <Input
+                name="address"
+                value={output.address}
+                placeholder="..."
+                onChange={e => this.handleChange(e, index)}
+              />
+            </FormGroup>
+            <FormGroup>
+              <Label>Amount</Label>
+              <Input
+                name="btc"
+                value={output.btc}
+                type="number"
+                step="0.00000001"
+                placeholder="..."
+                onChange={e => this.handleChange(e, index)}
+              />
+            </FormGroup>
+          </div>
+        )}
+        <Button color="primary" size="lg" block onClick={this.handleAddOutput}>
+          Add Output
+        </Button>
+        <Button color="primary" size="lg" block>
+          Submit
+        </Button>
+      </Form>
+    );
+  }
+
+  private handleChange = (ev: React.ChangeEvent<HTMLInputElement>, index: number) => {
+    let { outputs } = this.state;
+    let target;
+    if (ev.currentTarget.name === 'btc') {
+      target = Number(ev.currentTarget.value);
+    } else {
+      target = ev.currentTarget.value;
+    }
+    (outputs[index] as any)[ev.currentTarget.name] = target;
+    this.setState({ outputs });
+  };
+
+  private handleAddOutput = (ev: React.FormEvent<HTMLInputElement>) => {
+    let { outputs } = this.state;
+    outputs.push({ address: undefined, btc: undefined });
+    this.setState({ [ev.currentTarget.name]: ev.currentTarget.value } as any);
+  };
+
+  private handleSubmit = async (ev: React.FormEvent<HTMLFormElement>) => {
+    ev.preventDefault();
+    try {
+      this.setState({ isSubmitting: true });
+      const { outputs } = this.state;
+      const { activeWallet } = this.props;
+      await api.createPSBT({
+        wallet_name: activeWallet ? activeWallet.name : '',  // FIXME
```

Comment:
the `activeWallet: Wallet | null` forces me to do this ..

Reply:
Could also do `if (!activeWallet) return;` at the beginning, or the throw selector I mentioned.